docs(how-to): database instances with private networks (#224)
scaleway/docs-content: faq/database.mdx

Currently, you can connect one Private Networks on each Database Instance.

## Can I use any IP Range for my Private Network? 

For now, only the [RFC1918](https://datatracker.ietf.org/doc/html/rfc1918) IP subnets are accepted as Database Instances virtual IPv4.

## How is the Database private failover IP is managed?

The virtual IPv4 used to expose the Database Instance service is automatically routed to the new promoted node and announced within 10s to the neighbor Compute Instances.

## Can I remove the LoadBalancer Endpoint from my Database Instance? 
You can not remove the LoadBalancer Endpoint from your Database Instance, but you can block all access by emptying the allowed IPs list from the console or via the API.

## Can I limit the IPs allowed to reach my Database Instance on a Private Network?

It is not possible to limit access to a Database Instance on a Private Network. We recommend to create a dedicated Private Network for your Database Instances and add only the relevant middleware application Instances to this Private Network.

## Can I add an IPv6 to my Database Instance?

IPv6 is not supported with Private Networks on Database Instances.

> ## Where is Private Network available for Database Instances?

Reviewer: ```suggestion
## In what regions is the Private Networks feature available for Managed Databases?
```

> Private Network is available for Database Instances in PAR1, AMS1 and WAW1.

Reviewer: ```suggestion
Private Networks is available for Database Instances in PAR1, AMS1 and WAW1.
```

> Currently, you can not connect PAR2 or PAR3 Instances to a Database Instance located in the **Paris** region using Private Network.

Reviewer: ```suggestion
Currently, you cannot connect PAR2 or PAR3 Instances to a Database Instance located in the **Paris** region using Private Networks.
```
